src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+import java.util.ArrayList;
+
+// Event class representing an event, with fields course, teacher, date and time
+public class Event {
```

Comment:
Right now, this design tells us to instantiate Event anywhere in the program since it isn't used by any other class. Maybe we can put an Event pointer into the Department class so it becomes clear, that department will be organizing the events.

Reply:
done

Reply:
PLease push the changes